Python/compile.c

```diff
@@ -2235,7 +2243,6 @@ compiler_class(struct compiler *c, stmt_ty s)
                 compiler_exit_scope(c);
                 return ERROR;
             }
-            assert(i == 0);
```

Comment:
This is asserting that class scopes cannot have any cells other than the `__class__` cell. With inlining of comprehensions in class scopes, this is just not true anymore, so the assertion is not relevant; there is no other assertion to replace it with.

As far as I've been able to discover, there is no actual problem with cells in class scopes (as evidenced by the existence of `__class__` cells), it's just something that couldn't previously occur, and so it made sense to sanity-check it. Now it can occur.